### ISSUE TITLE
Dash video segment SIDX has an incorrect subsegment duration=0 values

### ISSUE THREAD

**Issue by Murmur** [gpac/gpac#729]
(Version: MP4Box - GPAC version 0.6.2-DEV-rev1258-gb731e76-master, Win10-64bit)

This command creates 12sec segment.m4s files, each have six subsegments in SIDX table (12s/2s). SIDX table has timescale=24000, subsegments 1..5 duration=0, subsegment 6 duration=288000. This is introduced in each segment.m4s file. 

Is this a bug have subsegduration=0. Should each SIDX subsegment duration value be written as 48000 value? 
Known bug in mp4box should I use `-dash 12000 -frag 12000` so just one subsegment is written?

```
MP4Box -noprog -dash 12000 -frag 2000 -mem-frags -rap -profile dashavc264:live ^
 -bs-switching no -sample-groups-traf -single-traf ^
 -url-template -segment-timeline -segment-name "test_$RepresentationID$_$Number$$Init=init$" ^
 -out "test.mpd" ^
 "temp-360p.mp4#trackID=1:id=v1" ^
 "temp-720p.mp4#trackID=1:id=v2" ^
 "temp-1080p.mp4#trackID=1:id=v3" ^
 "temp-128k.mp4#trackID=1:id=a4"
```

- - - -
(bonus round) Manifest media template mask has $Time$ variable but was expecting to see "test_$RepresentationID$_$Number$.m4s" value. Segment files are written as a test_v1_1.m4s, test_v1_2.m4s, test_v1_3.m4s, ...

**edit, ignore this bonus round** I had a `-segment-timeline` in a commandline but without it I get url template as expected without $Time$ variable. 
```
  <AdaptationSet segmentAlignment="true" group="1" maxWidth="1024" maxHeight="426" maxFrameRate="24" par="1024:426" lang="und">
   <SegmentTemplate timescale="1000" media="test_$RepresentationID$_$Number$$Time$.m4s" startNumber="1" initialization="test_$RepresentationID$_init.mp4">
    <SegmentTimeline>
     <S t="0" d="12000" r="4"/>
    </SegmentTimeline>
   </SegmentTemplate>

```

**Comment by huangh0604** [comment]
I caught this issue too.

**Comment by jeanlf** [comment]
This should have been fixed in 0.9.0, please reopen if needed
